### ISSUE TITLE
Diversity of the dataset predicates ? 

### ISSUE THREAD

**Issue by hadyelsahar** [gabrielStanovsky/oie-benchmark#3]
Due to having QA-SRL dataset as a source, most of the OIE benchmark predicates are single worded verbs. In Open IE systems output relations might include other forms such as ( Nouns or Noun+prep) Are there any future work considering that ? (for ex: expanding the dataset with other available annotations such as [AMR](https://link.springer.com/chapter/10.1007/978-3-319-46547-0_2) ) 

Thanks for the benchmark, great work!.

**Comment by gabrielStanovsky** [comment]
@hadyelsahar 
That's definitely a property of the data set and the conversion method.
This is somewhat mitigated in the soft comparison metric we use when comparing OIE systems which usually differ.

I'm definitely interested in expanding this resource with more annotations. 
One issue with using AMR for this purpose is that we don't have an annotation of the alignment between AMR concepts and the surface words.

BTW, if it's okay with you I might move this discussion to a wiki page, or if elsewhere if you have a better suggestion, as it's not really a short-term issue :)

Thanks!

**Comment by gabrielStanovsky** [comment]
BTW, @hadyelsahar, I just published a new version of the corpus which incorporates words from the question into the predicate, thus creating more diverse multi-word predicates. For example - the following predicates are created:
```
can be described as
can be equipped with
```
You can see the full predicate list [here](https://raw.githubusercontent.com/gabrielStanovsky/oie-benchmark/master/predicate_list.txt).

The [comparisons of the different extractors](https://github.com/gabrielStanovsky/oie-benchmark/blob/master/eval/eval.png) do not change much following this change.

**Comment by hadyelsahar** [comment]
Thanks for handling that very quickly. Would be very interested to know more about how did u collect those new predicates. Or they are the same as the old ones with prepositions attached when possible ?

**Comment by gabrielStanovsky** [comment]
These come from the same QA-SRL dataset, but I've taken some care to pull out elements from the associated questions (auxiliaries,  copulas, negations, prepositions) to form multi-word predicates, where applicable. 

For instance, in certain cases, the prepositions in questions actually belong in the argument rather than in the predicate.
Compare the following two automatic extractions from the corpus (predicate before arguments, in parentheses):

1.
```
China 's parliament ousted two Hong Kong residents *from* a panel drafting a new constitution for the colony .    
ousted  (China 's parliament, two Hong Kong residents,  *from* a panel drafting a new constitution for the colony China 's parliament)
```

2.
```
The borrowing to raise these funds would be paid off as assets of sick thrifts are sold .       
would be paid off  (The borrowing to raise these funds, as assets of sick thrifts are sold)
```

I hope these make sense.

In summary - this admittedly still misses some diversity in predicates, but I think it extends the previously mostly single-word predicates.
